purchases/src/main/kotlin/com/revenuecat/purchases/galaxy/GalaxyBillingWrapper.kt

```diff
@@ -347,13 +356,50 @@ internal class GalaxyBillingWrapper(
 
     override fun isConnected(): Boolean = true
 
+    @OptIn(GalaxySerialOperation::class)
     override fun queryPurchases(
         appUserID: String,
         onSuccess: (Map<String, StoreTransaction>) -> Unit,
         onError: (PurchasesError) -> Unit,
     ) {
-        warnLog { "Unimplemented: GalaxyBillingWrapper.queryPurchases" }
-        onSuccess(emptyMap())
+        log(LogIntent.DEBUG) { RestoreStrings.QUERYING_PURCHASE }
+        serialRequestExecutor.executeSerially { finish ->
+            getOwnedListHandler.getOwnedList(
+                onSuccess = { ownedProducts ->
+                    val storeTransactions = ownedProducts
+                        .filter {
+                            // TO DO: Find out what this returns for OTPs when we support OTPs
+                            it.subscriptionEndDate.parseDateFromGalaxyDateString() > dateProvider.now
+                        }
+                        .map {
+                            try {
+                                it.toStoreTransaction(purchaseState = PurchaseState.UNSPECIFIED_STATE)
```

Comment:
Just to check, is there a concept of pending purchases in the galaxy store? And if so, there is no way to set it? Just wondering if there is a way to avoid using `UNSPECIFIED_STATE` and just default to `PURCHASED`.